pink/configuration.py

```diff
@@ -106,15 +106,18 @@ def update(self, q: Optional[np.ndarray] = None) -> None:
         pin.computeJointJacobians(self.model, self.data, self.q)
         pin.updateFramePlacements(self.model, self.data)
 
-    def check_limits(self, tol: float = 1e-6) -> None:
+    def check_limits(self, tol: float = 1e-6, safe_break: bool = True) -> None:
         """Check that the current configuration is within limits.
 
         Args:
-            tol: Tolerance in radians.
+            tol (float): Tolerance in radians.
```

Comment:
```suggestion
            tol: Tolerance in radians.
```
Sphinx already parses type annotations via the `sphinx_autodoc_typehints` plugin (for instance the type of `tol` here is properly rendered [here in the docs](https://stephane-caron.github.io/pink/introduction.html#pink.configuration.Configuration.check_limits)), so we don't need to manually repeat them.